src/UI/Actions.hs

```diff
@@ -507,10 +508,13 @@ invokeEditor' s = do
   let editor = view (asConfig . confEditor) s
   tmpdir <- getTemporaryDirectory
   tmpfile <- emptyTempFile tmpdir "purebred.tmp"
-  status <- system (editor <> " " <> tmpfile)
+  status <- onException (system (editor <> " " <> tmpfile)) (pure $ setError editorError)
   case status of
-    ExitFailure _ -> pure $ set asAppMode BrowseMail s -- ^ show error XXX
-    ExitSuccess -> pure $ set asAppMode ComposeEditor s & asCompose . cTmpFile ?~ tmpfile -- ^ go to compose editor
+    ExitFailure _ -> pure $ s & set asAppMode BrowseMail & setError editorError
+    ExitSuccess -> pure $ set asAppMode ComposeEditor s & asCompose . cTmpFile ?~ tmpfile -- go to compose editor
+
+editorError :: Error
+editorError = GenericError "Editor command exited with error code. Check the your editor configuration and your terminal."
```

Comment:
"Check the your editor" -> "Check your editor".

Also could you wrap this long line?

Reply:
Many thanks! Done